### PR TITLE
Update Shoot and Share README

### PR DESCRIPTION
**Pull request by danielpassos**
_Consent screen_ now is required to create a `Client ID`

### CODE REVIEW

**Comment by danielpassos** [comment]
@matzew @secondsun Could you review that?

File: ShootAndShare/README.md [hunk 1]
```diff
@@ -37,18 +37,25 @@ Now you need to enable the Drive API. To do that, navigate to APIs & auth > APIs
...
 ![Google Console - Enable APIs](docs/google-console-2.png)
 
-Now you need create new credentials to access your Drive accounts from the app. Go to APIs & auth > Credentials and inside OAuth area click the blue `Create new Client ID` button. This launches the following popup, where you select `Installed application` and `Other`:
+Now you need create new credentials to access your Drive accounts from the app. Go to APIs & auth > Credentials and inside OAuth area click the blue `Create new Client ID` button. 
 
-![Google Console - Credentials](docs/google-console-3.png)
+![Google Console - Create client ID](docs/google-console-3.png)
 
-On the last screen finally get to see the actual value of the generated `client id`, `secret id` and `redirect uris`, which you will use later:
+_consent screen_ is required so, click the blue `Configure consent screen`
```
Comment (secondsun) [inline ShootAndShare/README.md]: How about 
"You will need to create a consent screen.  Click the blue...

File: ShootAndShare/README.md [hunk 2]
```diff
@@ -37,18 +37,25 @@ Now you need to enable the Drive API. To do that, navigate to APIs & auth > APIs
...
 
-![Google Console - New credentials created](docs/google-console-4.png)
+![Google Console - Consent screen](docs/google-console-4.png)
+
+Select your email and fill the name of your app. After save that you will return to the Client ID
```
Comment (secondsun) [inline ShootAndShare/README.md]: and fill *in* the name

Comment (secondsun) [inline ShootAndShare/README.md]: Try "Afterwards, click save and you will return to the Client ID"

File: ShootAndShare/README.md [hunk 3]
```diff
@@ -37,18 +37,25 @@ Now you need to enable the Drive API. To do that, navigate to APIs & auth > APIs
...
 
-You also edit the 'Consent screen'. Go to API > Consent screen and edit the project informations.
+![Google Console - Create client ID](docs/google-console-5.png)
 
-![Google Console - New credentials created](docs/google-console-5.png)
+Select `Installed application` and `Other`:
 
+![Google Console - Credentials](docs/google-console-6.png)
+
+On the last screen finally get to see the actual value of the generated `client id`, `secret id` and `redirect uris`, which you will use later:
```
Comment (secondsun) [inline ShootAndShare/README.md]: screen *we* finally

**Comment by danielpassos** [comment]
@secondsun Done!

**Pull request merged**